README.md

```diff
@@ -1 +1,43 @@
 # AWeber FlexBox Grid
+AWeber FlexBox Grid is born from AWeber's need to have a responsive and modern grid system to use on its website and web application. This is why we used FlexBox grid as the base for this project but using SCSS as a preprocessor language to provide maintainability and ease of development.
```

Comment:
`AWeber FlexBox Grid is a responsive modern grid system for creating structured layouts in both websites and web applications. Based on flexbox properties, it is a customized version of the infamous [Flexbox Grid](http://flexboxgrid.com/) and uses SCSS as a preprocessor language for to facilitate development and maintenance.`

`If you find the grid useful in your projects, please give us a star and/or contribute :)`